tests/TestClassInputFieldCheckboxRenderer.php

```diff
@@ -0,0 +1,124 @@
+ *
+ */
+
+declare (strict_types=1);
+
+namespace onOffice\tests;
+
+use DI\Container;
+use DI\ContainerBuilder;
+use onOffice\SDK\onOfficeSDK;
+use onOffice\WPlugin\Field\Collection\FieldsCollectionBuilderShort;
+use onOffice\WPlugin\Renderer\InputFieldCheckboxRenderer;
+use onOffice\WPlugin\SDKWrapper;
+use onOffice\WPlugin\Types\Field;
+use onOffice\WPlugin\Types\FieldsCollection;
+use onOffice\WPlugin\Types\FieldTypes;
+use onOffice\WPlugin\Utility\Logger;
+
+/**
+ * @runTestsInSeparateProcesses
+ * @preserveGlobalState disabled
+ */
+class TestClassInputFieldCheckboxRenderer
+	extends \WP_UnitTestCase
+{
+
+	/** @var SDKWrapperMocker */
+	private $_pSDKWrapperMocker = null;
+
+	/** @var FieldsCollectionBuilderShort */
+	private $_pFieldsCollectionBuilderShort = null;
+
+	/** @var Container */
+	private $_pContainer = null;
+
+	/**
+	 *
+	 * @before
+	 *
+	 */
+
+	public function prepare()
+	{
+		$pContainerBuilder = new ContainerBuilder;
+		$pContainerBuilder->addDefinitions(ONOFFICE_DI_CONFIG_PATH);
+		$this->_pContainer = $pContainerBuilder->build();
+
+		$fieldsResponse = file_get_contents
+		(__DIR__ . '/resources/ApiResponseGetFields.json');
+		$responseArrayFields = json_decode($fieldsResponse, true);
+		$this->_pSDKWrapperMocker = new SDKWrapperMocker();
+		$this->_pSDKWrapperMocker->addResponseByParameters
+		(onOfficeSDK::ACTION_ID_GET, 'fields', '', [
+			'labels' => true,
+			'showContent' => true,
+			'showTable' => true,
+			'language' => 'ENG',
+			'modules' => ['address', 'estate'],
+		], null, $responseArrayFields);
+
+		$this->_pContainer->set(SDKWrapper::class, $this->_pSDKWrapperMocker);
+		$this->_pContainer->set(Logger::class, $this->getMockBuilder(Logger::class)->getMock());
+
+		$this->_pFieldsCollectionBuilderShort = $this->getMockBuilder(FieldsCollectionBuilderShort::class)
+			->setMethods(['addFieldsAddressEstate', 'addFieldsSearchCriteria'])
+			->setConstructorArgs([new Container])
+			->getMock();
+
+		$this->_pFieldsCollectionBuilderShort->method('addFieldsSearchCriteria')
+			->with($this->anything())
+			->will($this->returnCallback(function (FieldsCollection $pFieldsCollection): FieldsCollectionBuilderShort {
+				return $this->_pFieldsCollectionBuilderShort;
+			}));
+
+		$this->_pFieldsCollectionBuilderShort->method('addFieldsAddressEstate')
+			->with($this->anything())
+			->will($this->returnCallback(function (FieldsCollection $pFieldsCollection): FieldsCollectionBuilderShort {
+				$pFieldMultiParkingLot = new Field('multiParkingLot', onOfficeSDK::MODULE_ESTATE);
+				$pFieldMultiParkingLot->setType(FieldTypes::FIELD_TYPE_INTEGER);
+				$pFieldMultiParkingLot->setLabel('Parking');
+				$pFieldsCollection->addField($pFieldMultiParkingLot);
+
+				return $this->_pFieldsCollectionBuilderShort;
+			}));
+
+		$this->_pContainer->set(FieldsCollectionBuilderShort::class, $this->_pFieldsCollectionBuilderShort);
+	}
+
+	/**
+	 *
+	 */
+	public function testRenderMultiParkingLotField()
```

Comment:
I like this test, do you think it is possible to test a 'regular' field as well?